adobe/spectrum-web-components
# Expand `slottable-request` usage and APIs

## Description
- allow `open` as a flag in the Overlay directives
- abstract `slottable-request` down for use with non-Overlay elements
- prioritize non-`type="hint"` Overlays on the same trigger
- expand options for performance testing specific files
- clean up performance test files
- add `slottable-request` and associated testing to the Action Menu

### To Dos:
- [x] test the new directive
- [x] test the `open` flag
- [x] meet coverage levels
- [x] confirm story names and update VRT cache

## How has this been tested?
    1. Go [here](https://slottable-action-menu--spectrum-web-components.netlify.app/storybook/?path=/story/action-menu--decorated)
    2. Open and close the menu
    3. See that it works roughly like it did before
    1. Go [here](https://slottable-action-menu--spectrum-web-components.netlify.app/storybook/?path=/story/overlay-directive--default)
    2. See that toggling `open` in the Controls pane opens and closes the Popover
    1. Go [here](https://slottable-action-menu--spectrum-web-components.netlify.app/storybook/?path=/story/tooltip-directive--default)
    2. See that toggling `open` in the Controls pane opens and closes the Tooltip

## Types of changes

## Checklist
